### DISCUSSION TITLE
IDE Support (PhpStorm)

### DISCUSSION THREAD

**Discussion by Sakuya** [gravitational/teleport#15205] [Q&A]
Hello. 
I encountered a problem, there is no way I can connect PhpStorm IDE to connect to servers remotely for development.
VSCode works in this regard and no complaints. 

Can you please tell me what to do.

When trying to add a connection, PhpStorm reports this:
"Connection to 'server.bastion' failed. Request failed"
" WARN - #c.j.p.w.c.RemoteConnection - Retrying connecting to sftp://server.bastion:22/ in @67262bd8: 1 iterations"
"org.apache.commons.vfs2.FileSystemException: Could not resolve file "sftp://gitlab.bastion/".

- Server: Teleport v10.1.2 git:v10.1.2-0-g5bc7caf go1.18.3
- Client: Teleport v10.0.2 git:v10.0.2-0-g47e0914 go1.18.3
Proxy version: 10.1.2

**Comment by webvictim** [comment]
I'm unclear how it's trying to use SFTP here but it might not be fully supported by Teleport yet.

**Reply by gistol** [reply]
I've got the same problem on PhpStorm 2021.3.1
I hope it will be fixed soon.
